example.py

```diff
@@ -25,7 +25,7 @@ def uav_example():
             # To access specific sensor data:
             pixels = state["RGBCamera"]
             velocity = state["VelocitySensor"]
-            # For a full list of sensors the UAV has, view the README
+            # For a full list of sensors the UAV has, view the docs
```

Comment:
For a full list of sensors the UAV has, consult the configuration file "InfiniteForest-MaxDistance.json"